### PR TITLE
GH-24: As a developer I need tagged template literals

### PR DESCRIPTION
**Pull request by mor-n4**
See #24.

### CODE REVIEW

**Comment by mor-n4** [comment]
Builds green.

File: plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/validators/UnsupportedFeatureValidator.xtend [hunk 1]
```diff
@@ -85,7 +85,7 @@ class UnsupportedFeatureValidator extends AbstractN4JSDeclarativeValidator {
 	// (see method ExpressionTransformation#transformOptionalChaining(ExpressionWithTarget))
 	@Check
 	def void checkTaggedTemplateLiteral(TaggedTemplateString tts) {
-		unsupported("tagged template literals", tts, N4JSPackage.Literals.EXPRESSION_WITH_TARGET__TARGET);
+		//unsupported("tagged template literals", tts, N4JSPackage.Literals.EXPRESSION_WITH_TARGET__TARGET);
```
Comment (mmews-n4) [inline plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/validators/UnsupportedFeatureValidator.xtend]: Why leaving it in the code?

**Comment by mor-n4** [comment]
Thanks @mmews-n4! Good point about the tests for flow graphs; I added some.

File: tests/org.eclipse.n4js.flowgraphs.tests/model/fix_first/AstOrder_Expressions.n4js.xt [hunk 2]
```diff
@@ -161,18 +161,63 @@ function EqualityExpression() {
...
+	0: function FunctionExpression02() { (function() { return "result"; }); },
+	1: { (function() { return "result"; }); },
+	2: (function() { return "result"; });,
+	3: (function() { return "result"; }),
+	4: function() { return "result"; },
+	5: { return "result"; },
+	6: return "result";,
+	7: "result"
+--- */
+function FunctionExpression02() {
```
Comment (mmews-n4) [inline tests/org.eclipse.n4js.flowgraphs.tests/model/fix_first/AstOrder_Expressions.n4js.xt]: Thanks for adding also these tests!

**Pull request merged**